### DISCUSSION TITLE
Permissions on /var/lib/dovecot/db/shared-mailboxes.db needed changing to make shared mailboxes visible

### DISCUSSION THREAD

**Discussion by awnz** [Intermesh/groupoffice#1060] [General]
Hi there,

I'm evaluating a bunch of possible Zimbra OSE replacements for the homelab family mailserver as the time ticks down on the end of Zimbra OSE. Group-Office is in the lead :)

I did however have one issue when setting up a test VM, trying shared mailboxes didn't show them to other users. I traced this down to what appears to be incorrect ownership of /var/lib/dovecot/db/shared-mailboxes.db which got created owned by root:root (and size 0).

I had to change the ownersip to vmail:mail for both the db directory and shared-mailboxes.db file for shares to start becoming visible.

Has anyone else hit this issue, or did I somehow stuff up installation?

Thanks,

Andrew

**Comment by mschering** [comment]
Thank you for evaluating Group-Office! 

Did you set it up using the Debian packages or docker?

Best regards,
Merijn

**Comment by awnz** [comment]
Hi Merjin,

This was installed using the deb packages as the installation instructions said this is the preferred method. It's on Ubuntu 22.04.

Later on I can spin up another test instance to see if I can reproduce the issue.

I've just got one other outstanding issue around search first, but I'll start another thread for that.

Thanks,

Andrew

**Comment by mschering** [comment]
I've just did a fresh install on a Debian bookworm Vm and the permissions were correct:

<img width="1392" alt="image" src="https://github.com/Intermesh/groupoffice/assets/1314447/7e403d0b-a2f0-4c21-8415-25920796babd">

**Reply by awnz** [reply]
Ok - later on I'll see if I can recreate it. Probably just something I managed to screw up, I think I may have accidentally allocated only 1GB RAM to the VM at install time 🙃
